docusaurus/docs/tutorials/build-an-app-plugin.md

```diff
@@ -0,0 +1,186 @@
+---
+id: build-an-app-plugin
+title: Build an app plugin
+sidebar_position: 10
+description: Learn how to create an app plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - app
+---
+
+import CreatePlugin from '@shared/create-plugin-frontend.md';
+import PluginAnatomy from '@shared/plugin-anatomy.md';
+
+## Introduction
+
+In this tutorial you will learn how to create _app plugins_. App plugins are Grafana plugins that allow you to bundle data sources and panel plugins within a single package. They also enable you to create custom pages within Grafana, providing a dedicated space for documentation, sign-up forms, or integration with other services via HTTP.
+
+App plugins are displayed in the navigation menu and offer the flexibility to define [UI Extensions](../ui-extensions/).
```

Comment:
Are they always?

Reply:
It would be useful for some further explanation here. At the start we say app plugins allow you to bundle data sources and panel plugins. If that's my intended use case, I probably don't want my app plugin to be displayed in the navigation menu; is this optional?

Reply:
I changed the wording to note it is optional